benchmark/benchmark.py

```diff
@@ -20,7 +20,7 @@
 def print_entry(name, res):
     print(" - ", name)
     print(" " * 4, end="")
-    for name, dt in res.items():
+    for name, dt in res.items():  # noqa: PLR1704
```

Comment:
What's this rule?

Reply:
% [`ruff rule PLR1704`](https://docs.astral.sh/ruff/rules/redefined-argument-from-local)
# redefined-argument-from-local (PLR1704)

Derived from the **Pylint** linter.

## What it does
Checks for variables defined in `for`, `try`, `with` statements
that redefine function parameters.

## Why is this bad?
Redefined variables can cause unexpected behavior because of overridden function parameters.
If nested functions are declared, an inner function's body can override an outer function's parameters.

## Example
```python
def show(host_id=10.11):
    for host_id, host in [[12.13, "Venus"], [14.15, "Mars"]]:
        print(host_id, host)
```

Use instead:
```python
def show(host_id=10.11):
    for inner_host_id, host in [[12.13, "Venus"], [14.15, "Mars"]]:
        print(host_id, inner_host_id, host)
```

## Options
- `lint.dummy-variable-rgx`

## References
- [Pylint documentation](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/redefined-argument-from-local.html)

Reply:
Nice maybe we should rename the variable because that is bad.